src/charm.py

```diff
@@ -622,6 +629,104 @@ def _get_cpu_extensions() -> list[str]:
                 del cpu_flags[0]
         return cpu_flags
 
+    def _get_access_nad_config(self) -> Dict[Any, Any]:
+        """Get access interface NAD config.
+
+        Returns:
+            config (dict): Access interface NAD config
+
+        """
+        config = {
+            "cniVersion": "0.3.1",
+            "ipam": {
+                "type": "static",
+                "routes": [
+                    {
+                        "dst": self._get_gnb_subnet_config(),
+                        "gw": self._get_access_network_gateway_ip_config(),
+                    },
+                ],
+                "addresses": [
+                    {
+                        "address": self._get_access_network_ip_config(),
+                    }
+                ],
+            },
+            "capabilities": {"mac": True},
+        }
+        if self._get_access_interface_mtu_config():
+            config.update({"mtu": self._get_access_interface_mtu_config()})  # type: ignore
+        return config
+
+    def _get_core_nad_config(self) -> Dict[Any, Any]:
+        """Get core interface NAD config.
+
+        Returns:
+            config (dict): Core interface NAD config
+
+        """
+        config = {
+            "cniVersion": "0.3.1",
+            "ipam": {
+                "type": "static",
+                "addresses": [
+                    {
+                        "address": self._get_core_network_ip_config(),
+                    }
+                ],
+            },
+            "capabilities": {"mac": True},
+        }
+        if self._get_core_interface_mtu_config():
+            config.update({"mtu": self._get_core_interface_mtu_config()})  # type: ignore
```

Comment:
```suggestion
        if core_mtu := self._get_core_interface_mtu_config():
            config.update({"mtu": core_mtu})
```

In addition to fixing the type hint for the `_get_core_interface_mtu_config()` method, this should work.

Reply:
OK